e2e/snapshot-formatting-changes/__tests__/snapshot.test.js

```diff
@@ -13,14 +13,14 @@ describe('snapshot serializer', () => {
       array: [{hello: 'Danger'}],
     };
     expect(object).toMatchInlineSnapshot(`
-{
-        "array": [
-                {
-                        "hello": "Danger",
-                },
-        ],
-}
-`);
+      {
+              "array": [
+                      {
+                              "hello": "Danger",
+                      },
+              ],
+      }
+    `);
```

Comment:
jest agrees with this new indentation if regenerated, fwiw. possibly fixed back in https://github.com/jestjs/jest/pull/11560 or something